src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/CourtController.kt

```diff
@@ -124,14 +103,22 @@ class CourtController(
     date: LocalDate,
 
     @ApiParam(
-      value = "The identifier for a court.  If present the response will only contain video link bookings for this court. Otherwise all bookings will be returned.",
+      value = "The name a court.  If present the response will only contain video link bookings for this court. Otherwise all bookings will be returned.",
       required = false,
-      example = "Wimbledon"
+      example = "Wimbledon",
     )
     @RequestParam(name = "court", required = false)
-    court: String?
+    court: String?,
+
+    @ApiParam(
+      value = "The identifier of a court.  If present the response will only contain video link bookings for this court. Otherwise all bookings will be returned. Takes precedence over court.",
```

Comment:
this will eventually change to be mandatory won't it? - as we'll be removing the all option

Reply:
Yes. Currently PSH can supply a court name as free text - so we have to store court names.  Once that feature is removed from PSH we can use court id exclusively